roles/openshift_control_plane/tasks/static.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Enable bootstrapping in the master config
+  yedit:
+    src: /etc/origin/master/master-config.yaml
+    edits:
+    - key: kubernetesMasterConfig.controllerArguments.cluster-signing-cert-file
+      value:
+      - /etc/origin/master/ca.crt
+    - key: kubernetesMasterConfig.controllerArguments.cluster-signing-key-file
+      value:
+      - /etc/origin/master/ca.key
+  when: openshift_master_bootstrap_enabled | default(False)
```

Comment:
This role is only included when openshift_master_bootstrap_enabled is true.  This boolean is redundant.